app/views/items/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<center>
+  <section id="item-title">
+    <% if @merchant %>
+    <h1><a href="/merchants/<%=@merchant.id%>"><%= @merchant.name %> Items</h1>
+    <% else %>
+      <h1>All Items</h1>
+    <% end %>
+  </section>
+
+<section class="boxes-container">
+  <% @items.each do |item| %>
+  <% if @merchant %>
+    <% if item.active? %>
+      <section class="single-box">
+        <div id="item-pic">
+          <%= link_to "/items/#{item.id}" do %>
+            <img src="<%=item.image %>" id="single-pic">
+          <% end %>
+        </div>
+        <div class="">
+          <p><%= link_to "#{item.name}", "/items/#{item.id}", method: :get %></p>
```

Comment:
since this link, where you're setting what the link display name will be, is already wrapped in erb, you can call item without interpolating. Like this: 
```html.erb
<%= link_to item.name, "/items/#{item.id}", method: :get %>
```
since `item.name` will return a string of that items name.